debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/legacy/MySqlConnectorTask.java

```diff
@@ -192,9 +191,8 @@ else if (!earliestBinlogFilename.endsWith("00001")) {
             if (startWithSnapshot) {
                 // We're supposed to start with a snapshot, so set that up ...
                 SnapshotReader snapshotReader = new SnapshotReader("snapshot", taskContext);
-                if (snapshotEventsAsInserts) {
```

Comment:
The equivalent change should also be done in the non-legacy `MySqlConnectorTask` class.

Reply:
I don't think we used that config option in non-legacy MySqlConnectorTask`.

Reply:
Ah, gotcha. That makes sense.